internal/termui/detect.go

```diff
@@ -46,6 +46,10 @@ func (d *Detect) Stop() {
 	d.doneChan <- true
 }
 
+func (d *Detect) SetNodes(map[string]*tview.TreeNode) {
+	// not implemented
```

Comment:
What will be happening here?

Reply:
Unfortunately that was me simply trying to satisfy the `type page interface`. That method is never intended to be implemented, but the _other page_ (Dashboard) does have it implemented.

Overall, I have serious misgivings the page orchestration. I planned to refactor but simply didn't come up with something I liked. If you'd prefer, I can spruce it up relatively soon.

```
type page interface {
	Handle(txt string)
	Stop()
	SetNodes(nodes map[string]*tview.TreeNode)
}
```

Reply:
Hmmm. I think it's ok to merge it in without it spruced up, given how it's hidden within an experimental flag and is going to be worked on more, but maybe add that in in a clarifying comment?

Reply:
I added clarifying comments.